app/src/common/shared/org/mozilla/vrbrowser/utils/DeviceType.java

```diff
@@ -2,25 +2,48 @@
 
 import android.util.Log;
 
+import androidx.annotation.IntDef;
+
 import org.mozilla.vrbrowser.BuildConfig;
 
 public class DeviceType {
     // These values need to match those in Device.h
+    @IntDef(value = { Unknown, OculusGo, OculusQuest, ViveFocusPlus, PicoNeo2, PicoG2 })
```

Comment:
It looks like `Device.h` needs to be updated?

Reply:
We need ViveFocus as well